rdt/transformers/null.py

```diff
@@ -121,6 +121,20 @@ def fit(self, data):
             if self._missing_value_generation == 'random':
                 self._null_percentage = null_values.sum() / len(data)
 
+    def _set_fitted_parameters(self, null_percentage):
```

Comment:
Could we change the parameter name to `null_ratio`? Or adjust the code so we have no confusion between being a percentage or a ratio (range of 0 to 100 vs 0 to 1) @frances-h, @npatki